core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1979,4 +1979,14 @@ private SqlLibraryOperators() {
           ReturnTypes.ARG0, null,
           OperandTypes.family(SqlTypeFamily.NUMERIC),
           SqlFunctionCategory.NUMERIC);
+
+  @LibraryOperator(libraries = {BIG_QUERY, ORACLE})
+  public static final SqlFunction EDIT_DISTANCE =
+      new SqlFunction("EDIT_DISTANCE",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.INTEGER_NULLABLE, null,
+          OperandTypes.family(ImmutableList.of(SqlTypeFamily.STRING, SqlTypeFamily.STRING,
+                  SqlTypeFamily.NUMERIC),
+              number -> number == 2),
+          SqlFunctionCategory.NUMERIC);
```

Comment:
can we use SqlTypeFamily.INTEGER instead of SqlTypeFamily.NUMERIC  as 3rd argument doesn't accept Decimal values.